app/models/finance/statement.rb

```diff
@@ -20,6 +21,11 @@ def past_deadline_date?
   def current?
     payment_date > Time.current && deadline_date > Time.current
   end
+
+  def identifier
```

Comment:
Could simplify this all bit with `#to_param` https://api.rubyonrails.org/classes/ActiveRecord/Integration.html#method-i-to_param:

```suggestion
  def to_param
```

Reply:
done